src/text.md

```diff
@@ -105,13 +105,15 @@ Some password managers will alert you if you have the same password for multiple
 > There are a few passwords you should memorize and maybe even keep out of your password manager. For example, you should always know the password to your main email address and the passwords to log in to your devices.
 
 ## Two-Factor Authentication (2FA)
-Two-factor authentication (sometimes called Multi-Factor Authentication, or MFA) provides an extra layer of security when you log in, and you should turn it on for every service that offers it.
+Two-factor authentication (sometimes called Multi-Factor Authentication, or MFA) provides an extra layer of security when you log in, and you should turn it on for every service that offers it. [^kalm-mfa]
+
+  [^kalm-mfa]: If you prefer to listen to audio explanations, Tom Merritt has a 9 minute podcast episode about Multi-Factor Authentication. [About Multi-Factor Authentication](https://dailytechnewsshow.com/2021/06/17/about-multi-factor-authentication/)
```

Comment:
I think it would work better to have `[a 9 minute podcast episode about Multi-Factor Authentication](...link...).` That way the link isn't just kinda hanging out there.